doc/classes/ProjectSettings.xml

```diff
@@ -469,7 +469,7 @@
 			[b]Note:[/b] This option is experimental and meant to alleviate stutter experienced by some users. However, some users have experienced a Vsync framerate halving (e.g. from 60 FPS to 30 FPS) when using it.
 		</member>
 		<member name="editor/script_templates_search_path" type="String" setter="" getter="" default="&quot;res://script_templates&quot;">
-			Search path for project-specific script templates. Script templates will be search both in the editor-specific path and in this project-specific path.
+			Search path for project-specific script templates. Script templates will be searched both in the editor-specific path and in this project-specific path.
```

Comment:
This reads weirdly, we may as well improve this sentence by using the active voice.

```suggestion
			Search path for project-specific script templates. Godot will search for script templates both in the editor-specific path and in this project-specific path.
```